sql/hive/src/test/scala/org/apache/spark/sql/hive/client/HiveClientVersions.scala

```diff
@@ -25,10 +25,7 @@ private[client] trait HiveClientVersions {
   private val testVersions = sys.env.get("SPARK_TEST_HIVE_CLIENT_VERSIONS")
   protected val versions = if (testVersions.nonEmpty) {
     testVersions.get.split(",").map(_.trim).filter(_.nonEmpty).toIndexedSeq
-  } else if (SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_9)) {
-    IndexedSeq("2.0", "2.1", "2.2", "2.3", "3.0", "3.1")
   } else {
-    IndexedSeq("0.12", "0.13", "0.14", "1.0", "1.1", "1.2", "2.0", "2.1", "2.2", "2.3", "3.0",
-      "3.1")
+    IndexedSeq("2.0", "2.1", "2.2", "2.3", "3.0", "3.1")
```

Comment:
Does this mean Spark drops support for HMS below 2.0 officially? Should we drop `HiveShim`s then? e.g. `Shim_v0_12` `Shim_v0_13`

Reply:
Yeah. I mean, it doesn't work with the JDK we supported so we should take it as dropping them.